spec/docs/participant_declarations_spec.rb

```diff
@@ -5,6 +5,13 @@
 RSpec.describe "Participant Declarations", type: :request, swagger_doc: "v1/api_spec.json" do
   let(:early_career_teacher_profile) { create(:early_career_teacher_profile) }
   let(:cohort) { early_career_teacher_profile.cohort }
+  let!(:school_cohort) { create(:school_cohort, school: early_career_teacher_profile.school, cohort: early_career_teacher_profile.cohort) }
```

Comment:
This shouldn't work I don't think - `create(:early_career_teacher_profile)` already creates a SchoolCohort, and I thought SchoolCohorts were unique on cohort&school - maybe not.
Either way this should be 
```suggestion
  let!(:school_cohort) { early_career_teacher_profile.school_cohort }
```

Reply:
They weren't linking up with the partnership, which is where we were seeing issues and why it had to become explicit either one way or the other.

Reply:
I'll try with this, since the issue that I was having was a disjoin between the mentor and ect profile school_cohorts and fixing one may fix the other.